slothy/targets/arm_v81m/cortex_m55r1.py

```diff
@@ -539,6 +541,7 @@ def get_min_max_objective(slothy):
         vmulf_T2,
         vfma,
     ): 2,
+    vmsr_FPSCR: 4,
```

Comment:
This is not what I see in the M55 SWOG. It says latency 1 and throughput 1. 
Where do you get the 4 from?